draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1043,8 +1071,14 @@
    For each stored response identified for update, the cache &MUST; use the
    header fields provided in the <x:ref>304 (Not Modified)</x:ref> response
    to replace all instances of the corresponding header fields in the stored
-   response.
+   response, with the following exceptions:
 </t>
+<ul>
+   <li>The exceptions to header field storage in <xref
+   target="storing.fields"/> also apply to header field updates.</li>
+   <li>Caches &MUST-NOT; update the following header fields:
+   Content-Encoding, Content-Length, Content-MD5, Content-Range, ETag.</li>
+</ul>
```

Comment:
Link the definitions? (I can update the PR)

Reply:
Please do!

Reply:
Done.